source/CommonJobs/CommonJobs.Application.EvalForm/EmployeeSearching/EmployeeToEvaluate_Search.cs

```diff
@@ -0,0 +1,96 @@
+﻿using CommonJobs.Domain.Evaluations;
+using Raven.Abstractions.Indexing;
+using Raven.Client.Indexes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.EmployeeSearching
+{
+    public class EmployeeToEvaluate_Search : AbstractMultiMapIndexCreationTask<EmployeeToEvaluate_Search.Projection>
+    {
+
+        public class Projection
+        {
+            public string Id { get; set; }
+            public string UserName { get; set; }
+            public string FullName { get; set; }
+            public string CurrentPosition { get; set; }
+            public string Seniority { get; set; }
+            public string Period { get; set; }
+            public string[] Evaluators { get; set; }
+            public string Responsible { get; set; }
+            public bool AutoEvaluationDone { get; set; }
+            public bool ResponsibleEvaluationDone { get; set; }
+            public bool CompanyEvaluationDone { get; set; }
+            public bool OpenToDevolution { get; set; }
+            public bool Finished { get; set; }
+        }
+
+        public EmployeeToEvaluate_Search()
+        {
+            AddMap<EmployeeEvaluation>(evaluations =>
+                from evaluation in evaluations
+                select new
+                {
+                    Id = evaluation.Id,
+                    UserName = evaluation.UserName,
+                    FullName = evaluation.FullName,
+                    CurrentPosition = evaluation.CurrentPosition,
+                    Seniority = evaluation.Seniority,
+                    Period = evaluation.Period,
+                    Evaluators = new dynamic[0],
+                    Responsible = evaluation.Responsible,
+                    AutoEvaluationDone = false,
+                    ResponsibleEvaluationDone = false,
+                    CompanyEvaluationDone = false,
+                    OpenToDevolution = evaluation.OpenToDevolution,
+                    Finished = evaluation.Finished
+                });
```

Comment:
The thing is we don't have the enum before that projection (we have the calification type and if it's finished or not), we fetch the users and create the enum for the DTO. And it seemed easy to do it that way but I'm new to this Raven projections so any advise is welcome!